.github/workflows/third-party.yaml

```diff
@@ -0,0 +1,53 @@
+name: Update third-party rules
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 1"
```

Comment:
Totally open to changing this.

Reply:
How about every 12 hours to start with? 

That way we don't get to where we accidentally issue a release with week-old rules.

Reply:
Updated in [`f0e9393` (#557)](https://github.com/chainguard-dev/malcontent/pull/557/commits/f0e9393990af2a87c4dd066f06a0eb6f4399a690).